trinity/nodes/base.py

```diff
@@ -44,11 +50,11 @@ class Node(BaseService):
     def __init__(self, chain_config: ChainConfig) -> None:
         super().__init__()
 
-        self._db_manager = create_db_manager(chain_config.database_ipc_path)
+        self._db_manager = create_db_manager(Path(chain_config.database_ipc_path))
```

Comment:
Maybe it's better for `chain_config` to return a `Path`.

Reply:
That was my first take but I abandoned it after looking through these errors https://github.com/ethereum/py-evm/pull/800

Can work to resolve them though.